.gitignore

```diff
@@ -8,6 +8,8 @@ classes
 .gradle
 gradle/ext/
 /bin/
+.DS_Store
```

Comment:
In Mac_OS, .DS_Store is a file that stores custom attributes of its containing folder, such as the position of icons or the choice of a background image. No need to commit them.